### ISSUE TITLE
LeftCurly "nl" doesn't work on METHOD_DEF

### ISSUE THREAD

**Issue by nator** [checkstyle/checkstyle#112]
This rule seems to work for all other tokens (if, while, class definition) but doesn't catch method definitions with a curly brace on the same line. I can't see any changes that would lead to this issue, any ideas?

```
<module name="LeftCurly">
    <property name="option" value="nl"/>
</module>
```
